debugger/pythonExtension/gdb_sgx_plugin.py

```diff
@@ -17,8 +17,8 @@
 OE_ENCLAVE_HEADER_FORMAT = 'QQQQQ'
 OE_ENCLAVE_MAGIC_VALUE = 0x20dc98463a5ad8b8
 
-# The following is the offset of the 'debug' and
-# 'simulate' fields which must lie one after the other.
+# The following are the offset of the 'debug' and
+# 'simulate' flag fields which must lie one after the other.
```

Comment:
Can you also fix the ThreadData reference below? I cannot find such a structure in host\enclave.h

Reply:
Yes, I replaced all references or ThreadData in the plugin to ThreadBinding, the current name.